waku/discovery/waku_discv5.nim

```diff
@@ -211,6 +211,8 @@ proc findRandomPeers*(
   elif wd.predicate.isSome():
     discoveredRecords = discoveredRecords.filter(wd.predicate.get())
 
+  waku_discv5_discovered.inc(discoveredRecords.len)
```

Comment:
The only concern that I have is if we receive the same peer twice, we will update this metric both times.

Another alternative is increasing it in `addPeer()` if the origin is `PeerOrigin.Discv5` after passing the duplicate checks

Reply:
:thinking:  I thought we had another metric for that...

Reply:
Sounds good :) Thank you!